app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/AdditionalOptions.kt

```diff
@@ -99,8 +95,8 @@ fun AdditionalOptionsMenu(
 
             AdditionalOptionMenuState.RichTextEditing -> {
                 RichTextOptions(
-                    onRichTextHeaderButtonClicked = { onRichOptionButtonClicked(RichTextMarkdown.Header) },
-                    onRichTextBoldButtonClicked = { onRichOptionButtonClicked(RichTextMarkdown.Bold) },
+                    onRichTextHeaderButtonClicked = { onRichOptionButtonClicked(RichTextMarkdown.Bold) },
+                    onRichTextBoldButtonClicked = { onRichOptionButtonClicked(RichTextMarkdown.Header) },
```

Comment:
Is that intentional ? `Bold` for `RichTextHeaderButton` and `Header` for `RichTextBoldButton`